### DISCUSSION TITLE
When disabling and reenabling subscriptions old content returns

### DISCUSSION THREAD

**Discussion by andersnas** [EyevinnOSC/community#19] [General]
May be a bug or a feature, but when you start a FAST channel subscription and start a channel- and then you remove the same subscription you'd assume the channel is gone- but it is not. If I start the subscription again the channel reappears, still playing. Maybe you are not spinning down the POD?

**Comment by birme** [comment]
It is correct that we are not taking down any instances instantly you unsubscribe. Just to be cautious in case that as accidental thing. We will clean up "dangling" instances on a regular basis
